lib/utils/logDeprecation.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+const chalk = require('chalk');
+const memoizee = require('memoizee');
+
+const loggedDeprecations = {};
+
+function consoleLog(...args) {
```

Comment:
I think there's not much value in this function. Let's remove it

Reply:
Done